### DISCUSSION TITLE
Where to put static assets and how to access

### DISCUSSION THREAD

**Discussion by Onurfesci** [keycloakify/keycloakify#415] [Q&A] [answered]
Hi, I couldn't find anywhere in the documentation information for where to place static assets and how to access within the React app so that they are available both in Storybook and the Keycloak instance.

For example, I have a logo at `/public/images/logo.svg`. Running Storybook, it works when I have `<img src="/images/logo.svg" />`, but this of course doesn't work when I build and use the theme.

Should I be placing this image somewhere else? In `/public/keycloak-resources` maybe? And how do I access it? It there some field in the `kcContext` which will give me the path to my static assets?

It would be great if this could be included in the documentation too. :) Thanks!

**Comment by garronej** [comment, marked as answer]
Hi @Onurfesci,  

Here you go, I just added a new section in the documentation:  https://docs.keycloakify.dev/v/v9/importing-assets

**Reply by Onurfesci** [reply]
Thanks! 🙌
